scalar/json_test.go

```diff
@@ -15,6 +21,7 @@ func TestJSONSet(t *testing.T) {
 		{source: "{}", result: JSON{Value: []byte("{}"), Valid: true}},
 		{source: `"test"`, result: JSON{Value: []byte(`"test"`), Valid: true}},
 		{source: "1", result: JSON{Value: []byte("1"), Valid: true}},
+		{source: `json some data`, result: JSON{Value: []byte(`"json some data"`), Valid: true}},
```

Comment:
The source value is an unquoted string so not sure why we're treating it as a valid JSON